edflow/edflow

```diff
@@ -89,6 +89,10 @@ def main(opt):
     # Training
     if opt.train:
         if opt.checkpoint is not None:
+            if not os.path.exists(opt.checkpoint):
```

Comment:
sorry but this can not be merged because for tensorflow you explicitly have to specify a checkpoint path which itself is not a file (only `path + {.meta,.index,.data...}` are actual files on the system)

Reply:
Removed the checkpoint path check.